### ISSUE TITLE
tasks.hotosm.org: redirect from http to https

### ISSUE THREAD

**Issue by JanZerebecki** [hotosm/osm-tasking-manager2#777]
I don't know if this is the right place for this task. Please direct me to a better one if it is not.

The installation at tasks.hotosm.org currently redirects from http to https instead of the other way around. This breaks the "Edit with iD editor" button when requiring https for accessing openstreetmap.org as the resulting openstreetmap.org URL contains a reference to "http://tasks.hotosm.org/....gpx" which results in "Blocked loading mixed active content" as openstreetmap.org was loaded via https.

This whole task can be solved by
1) redirect from http to https on tasks.hotosm.org
2) change the "http://tasks.hotosm.org/....gpx" URL passed as an argument to openstreetmap.org to https
3) change the target of the edit button to https://openstreetmap.org instead of http.
Possibly 2 and 3 were already done for #693, but I didn't verify that by reading the code.

**Comment by scruss** [comment]
Yes, this sounds the same as the (now closed as duplicate) #795 

OSM really shouldn't be serving mixed content
